pkg/hintrunner/zero/zerohint_math_test.go

```diff
@@ -555,5 +555,106 @@ func TestZeroHintMath(t *testing.T) {
 				errCheck: errorTextContains("outside of the range [0, 2**250)"),
 			},
 		},
+		"UnsignedDivRem": {
+			{
+				operanders: []*hintOperander{
+					{Name: "value", Kind: fpRelative, Value: feltUint64(100)},
+					{Name: "div", Kind: fpRelative, Value: feltUint64(6)},
+					{Name: "q", Kind: uninitialized},
+					{Name: "r", Kind: uninitialized},
```

Comment:
See in the code that `q` and `r` are references to the `range_check_ptr` so you'll need to handle that differently here. Check `Assert250bits` test implementation as an example. You'll need to fix this on the remaining test cases as well